.moai/specs/SPEC-NO-HYBRID-001/research.md

```
@@ -0,0 +1,208 @@
+후보 4: hook handler (`internal/hook/*.go`)
+- 현재: 일부 핸들러가 multiple event type 처리
+- 평가: 검토 필요
+
+후보 5: SKILL.md 일부 (e.g., moai-workflow-project)
+- 현재: doc gen + lang init + template optimization 모두 통합
+- 평가: 통합 자체는 OK이나 도구 / 명령어 차원 분리 검토
+
+### 2.2 Format-based control vs Tool-based control
+
+**Anti-pattern (format-based)**:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)